modules/library/referencing/src/main/java/org/geotools/referencing/factory/URN_Parser.java

```diff
@@ -119,4 +121,9 @@ public static URN_Parser buildParser(final String urn) throws NoSuchAuthorityCod
         throw new NoSuchAuthorityCodeException(
                 Errors.format(ErrorKeys.ILLEGAL_IDENTIFIER_$1, type), "urn:ogc:def", type);
     }
+
+    private static boolean versionIsEmpty(Version urnVersion) {
+        if (urnVersion == null) return false;
```

Comment:
Works in the context where it's used, but it's odd. Drawing a parallel to strings, one normally checks if they are null or empty, e.g.: http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence-

I would do the same here, the end result should not change, but the code is less surprising.

Reply:
@aaime understood, will use this function